### DISCUSSION TITLE
Is there a way to get have “top-level” tasks in turborepo?

### DISCUSSION THREAD

**Discussion by iiian** [vercel/turborepo#3141] [Help] [answered]
Say I've got a task that needs to happen once per project and is super identical. For example, I need to run a Docker-push on an image generated per project. For each project in my repo, it's pretty clear that it's going to be `docker push ${SCOPE}`, where SCOPE is whatever project I'm in. I _could_ embed this under the 20 something projects in my repo by hand. I could write a script that trawls my subdirs for package.jsons while avoiding node_modules, and auto-populates it. But the I'm hoping I can just add a top level turbo command that will execute against every project.

Thoughts?

**Comment by petermekhaeil** [comment, marked as answer]
You can run tasks that exist in the root `package.json`:

https://turbo.build/repo/docs/core-concepts/monorepos/running-tasks#running-tasks-from-the-root

Does this help?

**Reply by iiian** [reply]
Yes, thanks I will give this a shot. Appreciate the reply!
